List.cpp

```diff
@@ -0,0 +1,207 @@
+//
+//  List.cpp
+//  List
+//
+//  Created by Conor Sweeney on 4/21/20.
+//  Copyright © 2020 csweeney. All rights reserved.
+//
+
+#include "List.h"
+#include <iostream>
+
+
+using namespace std;
+
+List::List(){
+}
+
+string List::lpop(string key) {
+    //I am not check for the key so it will throw a map exception if key does not exist
+    //thoughts?
+    deque<string> d = m.at(key);
```

Comment:
You can throw an exception of your own defined class rather than exposing the internals

Reply:
Check if the key holds a deque value before assigning it to a deque.